### ISSUE TITLE
Placeholders in the paths

### ISSUE THREAD

**Issue by artemave** [electron/electron-rebuild#137]
Hello,

Latest version (1.5.5) stores build artifacts in the directories with placeholders in names. Then electron runtime can't find them and nothing works.

![image](https://cloud.githubusercontent.com/assets/23721/21660072/bca2066a-d2c5-11e6-9ddc-6edb9f92478c.png)

![image](https://cloud.githubusercontent.com/assets/23721/21660165/3d556900-d2c6-11e6-9aa4-d4f693d6d123.png)

I am on node 6.7, running `electron-rebuild` from a script with the following params:

![image](https://cloud.githubusercontent.com/assets/23721/21660548/be1a3290-d2c7-11e6-9b4c-71dddc24688a.png)

**Comment by MarshallOfSound** [comment]
@artemave I'll look into this when I get a chance, it looks like some weird prebuilt issue 😕

**Comment by MarshallOfSound** [comment]
@artemave PR is up to handle this 👍

**Comment by artemave** [comment]
Merge?

**Comment by antonycourtney** [comment]
Hit this same issue myself today, also when trying to run electron-rebuild on sqlite3. Spent an hour or so digging in enough to understand the cause and was just about to file my own bug report when I discovered this issue.  As far as I can tell node-sqlite3 can not be built for electron using electron-rebuild until this issue is resolved.

As an interim workaround: I can confirm that reverting electron-rebuild to 1.4.0 will work with the latest electron (1.4.14). (I've only tested with electron-1.4.7. and 1.4.14, but I'd expect everything in between to work too).
